examples/ex1p.cpp

```diff
@@ -209,11 +210,6 @@ int main(int argc, char *argv[])
 
    a->FormLinearSystem(ess_tdof_list, x, *b, A, X, B);
 
-   if (myid == 0)
-   {
-      cout << "Size of linear system: " << A->Height() << endl;
-   }
-
```

Comment:
This is actually a "feature" -- currently the `Operator` sizes are the local ones. We can decide to make them global, but that should be a separate change. Given that the output is the same as "Number of finite element unknowns", I think it is simplest to just take this out.